common/scripted_score_values/wc_scripted_ai_acceptance_score_values.txt

```diff
@@ -0,0 +1,242 @@
+ROOT_likes_FROM_5_positive_per_5_opinion_chance = {
```

Comment:
LGTM, but shouldn't be...
```suggestion
ROOT_likes_FROM_5_positive_per_5_opinion_score = {
```

Reply:
IIRC, score is for event chances, chance is for AI acceptance

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/79677058-8d8cb180-81f5-11ea-894a-e0030c45247b.png)

</details>